frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/TeamSettings/TeamSettings.tsx

```diff
@@ -0,0 +1,208 @@
+import Button from "components/buttons/Button";
+// @ts-ignore
+import InputField from "components/forms/fields/InputField";
+import useTeamIdParam from "hooks/useTeamIdParam";
+import { ITeamConfig } from "interfaces/team";
+import { ITeamSubnavProps } from "interfaces/team_subnav";
+import React, { useCallback, useContext, useEffect, useState } from "react";
+import { useQuery } from "react-query";
+import teamsAPI, { ILoadTeamResponse } from "services/entities/teams";
+import configAPI from "services/entities/config";
+import { IConfig } from "interfaces/config";
+import { NotificationContext } from "context/notification";
+import { IApiError } from "interfaces/errors";
+import Spinner from "components/Spinner";
+import DataError from "components/DataError";
+import TeamHostExpiryToggle from "./components/TeamHostExpiryToggle";
+
+const baseClass = "team-settings";
+
+const HOST_EXPIRY_ERROR_TEXT = "Host expiry window must be a positive number.";
+
+const TeamSettings = ({ location, router }: ITeamSubnavProps) => {
+  const [UITeamExpiryEnabled, setUITeamExpiryEnabled] = useState(false); // default false until API response
+  const [UITeamHostExpiryWindow, setUITeamHostExpiryWindow] = useState<
```

Comment:
Is the "UI" prefix meant to signify that this holds the local state of the form data? If so, maybe `formDataHostExpiryEnabled` and `formDataHostExpiryWindow` would be clearer?